pandas/tseries/offsets.py

```diff
@@ -252,6 +253,8 @@ def apply_index(self, i):
                                       "applied vectorized".format(kwd=kwd))
 
     def isAnchored(self):
+        # TODO: Does this make sense for the general case?  It would help
+        # if there were a canonical docstring for what isAnchored means.
```

Comment:
@jreback : Thoughts?

Reply:
yeah to be honested I am not sure ``isAnchored`` is really necessary, but that's orthogonal